From 0de640dd834b6c01c4904e11d51f3a1406c89469 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Mon, 3 Apr 2017 20:34:07 +0200
Subject: [PATCH] Remove -lanl when WITH_ADNS is unset

Do not add -lanl to BROKER_LIBS for all Linux builds.
Indeed, -lanl is only needed for getaddrinfo_a which is only used in
_mosquitto_try_connect_step1 when WITH_ADNS is set

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 config.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/config.mk b/config.mk
index 6e369c2..44639d2 100644
--- a/config.mk
+++ b/config.mk
@@ -159,10 +159,6 @@ ifeq ($(UNAME),QNX)
 	LIB_LIBS:=$(LIB_LIBS) -lsocket
 endif
 
-ifeq ($(UNAME),Linux)
-	BROKER_LIBS:=$(BROKER_LIBS) -lanl
-endif
-
 ifeq ($(WITH_WRAP),yes)
 	BROKER_LIBS:=$(BROKER_LIBS) -lwrap
 	BROKER_CFLAGS:=$(BROKER_CFLAGS) -DWITH_WRAP
-- 
2.5.0

